custom/languages/en_UK.json

```diff
@@ -252,7 +252,8 @@
     "admin/group_sync": "Group Sync",
     "admin/group_sync_info": "You can configure the API to automatically update a user's website group when their integration group is changed. Simply enter the integration group name\/ID and the website group it will be synchronised with the below rules.",
     "admin/group_sync_logs": "Group Sync Changes",
-    "admin/group_sync_plugin_not_set_up": "In-game plugin is not set up",
+    "admin/group_sync_server": "Select a Minecraft server to get group sync groups from",
+    "admin/group_sync_plugin_not_set_up": "In-game plugin is not set up or no group sync server has been selected",
```

Comment:
Unfortunately this leaks onto a second line, better than being not fully specific imo.